app/controllers/documents_controller.rb

```diff
@@ -42,10 +42,12 @@ def create
     @document = Document.new resource_params
     authorize! :create, @document
     if @document.save
```

Comment:
Let's try to keep this consistent with media and narratives:
`
    if @building
      @document.buildings << @building
    elsif @person
      @document.people << @person
    end
`